datacapturing/src/main/java/de/cyface/datacapturing/PongReceiver.java

```diff
@@ -25,7 +25,7 @@
  * tell the caller, that the service is not running.
  *
  * @author Klemens Muthmann
- * @version 1.1.4
+ * @version 1.1.7
```

Comment:
You should add your name here.
```suggestion
* @author Armin Schnabel
 * @version 1.1.7
```